src/potential/grdseamount.c

```diff
@@ -410,6 +422,54 @@ GMT_LOCAL void grdseamount_gaussian_area_volume_height (double a, double b, doub
 	*z = (*V) / (*A);
 }
 
+GMT_LOCAL double poly_smt_func (double r) {
+	/* Polynomial radial function similar to a Gaussian */
+	return (pow ((1.0 + r) * (1.0 - r), 3.0) / (1.0 + pow (r, 3.0)));
```

Comment:
Can you confirm that this is the correct functional form? I want to double check because the function given in the PR proposal is different. (h(r) = (1+r)^3(1-r)^2/(1+r^3))

Reply:
Yikes,  OK I will fix but after an hour of zoom.  This explains why my updated plots in doc/script looks good (gmt math on the right equation) while the test plot seems to have too much difference with a Gaussian (using the flawed output).
OR, you could fix this as per the equation and run tests and problaby need to update the test/potential/? plot.

Reply:
I actually think that both the test and the docs might use the correct equation and it's just the PR description that had the wrong one. This is what the seamount would look like with h(r) = (1+r)^3(1-r)^2/(1+r^3):
![smt_poly](https://user-images.githubusercontent.com/14077947/107715874-45eb8080-6c74-11eb-9278-9aac4d5b2b8d.jpg)